helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobBuilder.java

```diff
@@ -298,31 +300,73 @@ public TemporaryJob deploy(final List<String> hosts) {
     return job;
   }
 
+  /**
+   * Set the {@link #image(String)} field for the Docker image to use in this job from the output of
+   * docker-maven-plugin (image_info.json on classpath) or dockerfile-maven-plugin
+   * (docker/image-name on classpath)
+   */
   public TemporaryJobBuilder imageFromBuild() {
     final String envPath = env.get("IMAGE_INFO_PATH");
     if (envPath != null) {
       return imageFromInfoFile(envPath);
-    } else {
-      final String name = fromNullable(env.get("IMAGE_INFO_NAME")).or("image_info.json");
```

Comment:
removed this `or()` as the first part of the condition is always false in this branch